securedrop/manage.py

```diff
@@ -148,16 +148,17 @@ def _make_password():
 def _add_user(is_admin=False):
     username = _get_username()
 
-    print("Note: Journalist passwords are now autogenerated.")
+    print("Note: Passwords are now autogenerated.")
     password = _make_password()
-    print("This journalist's password is: {}".format(password))
+    print("This {}'s password is: {}"
+          .format('admin' if is_admin else 'journalist', password))
 
     is_hotp = _get_yubikey_usage()
     otp_secret = None
     if is_hotp:
         while True:
             otp_secret = raw_input(
-                'Please configure your YubiKey and enter the secret: ')
+                "Please configure this admin's YubiKey and enter the secret: ")
```

Comment:
My comment [here](https://github.com/freedomofpress/securedrop/pull/2476#discussion_r146936772) also applies to this line. You can also just generically say "user" for simplicity if you prefer (either approach is fine for merge).